dockerbuild/run.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+# This is a temporary fix (bug workaround) until ssp 1.20 is in use
```

Comment:
Based on [this comment](https://simplesamlphp.org/docs/stable/simplesamlphp-changelog.html#version-1190) in the SimpleSAMLphp changelog, I didn't think there was going to be a version 1.20:
> Version 1.19.0
> Released 2021-01-21
> * This version will be the last of the 1.x branch and will provide a migration path to our new templating system, routing system and translation system.

Have you seen other evidence to the contrary, suggesting that there will be a 1.20?